libkbfs/mdserver_remote.go

```diff
@@ -570,8 +570,31 @@ func (md *MDServerRemote) MetadataUpdate(_ context.Context, arg keybase1.Metadat
 	return nil
 }
 
+// FoldersNeedRekey implements the MetadataUpdateProtocol interface.
+func (md *MDServerRemote) FoldersNeedRekey(_ context.Context,
+	requests []keybase1.RekeyRequest) error {
+	if md.squelchRekey {
+		md.log.Debug("MDServerRemote: rekey updates squelched for testing")
```

Comment:
Please use `CDebugf(ctx,` here and below (`FolderNeedsRekey` should do that too, but you don't need to fix it in this PR).